src/components/sections/compensation-calculator/CompensationCalculator.tsx

```diff
@@ -0,0 +1,111 @@
+import Link from "next/link";
+import { Suspense } from "react";
+
+import LinkButton from "src/components/linkButton/LinkButton";
+import Text from "src/components/text/Text";
+import { getHref } from "src/utils/link";
+import { LocaleDocument } from "studio/lib/interfaces/locale";
+import { CompensationCalculatorSection } from "studio/lib/interfaces/pages";
+import { LOCALE_QUERY } from "studio/lib/queries/locale";
+import { loadStudioQuery } from "studio/lib/store";
+
+import { getHandbookLinksFromCompensationPage, getSalaryByYear } from "./api";
+import Calculator from "./Calculator";
+import styles from "./compensation-calculator.module.css";
+
+export interface CompensationCalculatorProps {
+  language: string;
+  section: CompensationCalculatorSection;
+}
+
+export default async function CompensationCalculator({
+  section,
+  language,
+}: CompensationCalculatorProps) {
+  const salariesRes = getSalaryByYear(2024, language);
```

Comment:
```suggestion
  const salariesRes = getSalaryByYear(new Date().getFullYear(), language);
```

Reply:
Yes, this was missing from my TODO. Was doing some handling of min/max/initial as we speak. out in a few

Reply:
But actually, this particular line I think we need to do more sophisticated so this is a temporary permanent hardcoded thing. Because we need to fetch "latest" page from Compensation page, to use that. I'll cover it in a separate PR